packages/markers/src/browser/marker-tree-model.ts

```diff
@@ -34,4 +34,12 @@ export class MarkerTreeModel extends TreeModelImpl {
     protected getOpenerOptionsByMarker(node: MarkerNode): OpenerOptions | undefined {
         return undefined;
     }
+
+    openNode(node: TreeNode): void {
```

Comment:
I would call it `revealNode` and it should not do anything if node is not `MarkerNode`.

You should not override `openNode` otherwise it breaks dbl click which should focus.

Reply:
Sure, sounds good.